src/delphi/eval/token_labelling.py

```diff
@@ -0,0 +1,184 @@
+"""
+This script creates labels for tokens in a sentence. 
+It takes the context of the token into account. 
+Additionally, it can visualize the sentences and their poart-of-speech (POS) tags.
+"""
+
+from pprint import pprint
+from typing import List, Optional
```

Comment:
you don't need to import `List` for typing, you can just use `list`

Reply:
You are right.
But unfortunately my venv was created using Python 3.8 where `List` is required.

> In Python 3.8 and earlier versions, to specify the type of elements within containers like lists, you need to import List from the typing module. This is because the built-in list type was not subscriptable (you couldn't use list[int] directly for type hints).

Reply:
please update to 3.10, you'll run into issues when we use features from 3.9+ (I think we already do?)

Reply:
LMK if you need help on this

Reply:
Done